pkg/docker-cli/Dockerfile

```diff
@@ -96,7 +96,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends apt-utils bash
 RUN <<EOT
   set -e
   case "$PKG_RELEASE" in
-    ubuntu2004|ubuntu2204|ubuntu2304)
+    ubuntu2004|ubuntu2204|ubuntu2304|ubuntu2310)
```

Comment:
Since `ubuntu2210` is skipped, should we be skipping `ubuntu2304` too?

Reply:
`ubuntu2210` is EOL. Was removed in https://github.com/docker/packaging/pull/136.